File: exercicios em C/q37.cpp
#include <stdio.h>

int main(void){
	int raio, h, area, base, altura, aresta, i, n, op, volume, soma;
	printf("\n\nEste programa soma o voume de diferentes tanques.");
	printf("\n\nDigite a quantidade de tanques que deseja somar os valores volume desejada.");
	scanf("%d", &n);
	for(i = 1; i <= n; i++){
		printf("\n\nMenu:");
		printf("\n\n1. Cubo.");
		printf("\n\n2. Paralelepipedo.");
		printf("\n\n3. Cilindro.");
		printf("\n\n4. Cone.");
		printf("\n\nDigite a opcao a qual deseja somar o volume: ");
		scanf("%d", &op);
		switch (op) {
			case 1:
			printf("\n\nVoce escolheu Cubo!");
			printf ("\n\nDigite da aresta do cubo: ");
			scanf ("%d", &aresta);
			volume = aresta * aresta *aresta;
			break;

			case 2:
			printf("\n\nVoce escolheu Paralelepipedo!");
			printf ("\n\nDigite o valor da base: ");
			scanf ("%d", &base);
			printf ("\n\nDigite o valor da altura: ");
			scanf ("%d", &altura);
			printf ("\n\nDigite o valor da area: ");
			scanf ("%d", &area);
			volume = area * altura * base;
			break;
	
			case 3:
			printf("\n\nVoce escolheu cilindro!");
			printf ("\n\nDigite o valor do raio: ");
			scanf ("%d", &raio);
			printf ("\n\nDigite o valor da altura: ");
			scanf ("%d", &h);
			volume = 3,14 * (raio * raio) * h;
			break;
	
			case 4:
			printf("\n\nVoce escolheu Cone!");
			printf ("\n\nDigite o valor do raio: ");
			scanf ("%d", &raio);
			printf ("\n\nDigite o valor da altura: ");
			scanf ("%d", &h);
			volume = (3,14 * ((raio * raio) * h)) / 3;
			break;
			
			default:
			printf("\n\nOpcao invalida!!!");
			break;
		}
		soma = soma + volume;
	}
	printf("\n\nA soma total dos volumes e: %d", soma);
	printf("\n\nObrigado por utilizar nosso sistema!");
	return 0;
}

File: exercicios em Python/q37.py
import os
print("\nEste programa calcula o volume de diferentes tanques, de diferentes formato, e soma todos!")
n = int(input("\nDigite o numero de unidade de tanque que ira somar o volume: "))
i = 1
volume = 0
soma = 0
while i <= n:
        print("\nMenu\n1.Cubo\n2.Paralelepipedo\n3.Cilindro\n4.Cone.")
        op = int(input("\nAgora digite o numero da opçao desejada: "))
        if op == 1:
                print("\n\t\tCubo!")
                aresta = float(input("\nDigite o valor da aresta do cubo: "))
                volume = int (aresta * aresta *aresta)
        
        elif op == 2:
                print("\n\t\tParalelepipedo!")
                areaB = float(input("\nDigite o valor da area da base: "))
                h = float(input("\nDigite o valor da altura: "))
                volume = int (areaB * h)

        elif op == 3:
                print("\n\t\tCilindro!")
                raio = float(input("\nDigite o valor do raio: "))
                h = float(input("\nDigite o valor da altura: "))
                volume = int(3.14 * (raio * raio) * h)

        elif op == 4:
                print("\n\t\tCone!")
                raio = float(input("\nDigite o valor do raio: "))
                h = float(input("\nDigite o valor da altura: "))
                volume = int((3.14 * ((raio * raio) * h)) / 3)

        else:
                print("\n\t\tOpçao invalida!")
                break

        soma = int(soma + volume)
        i = i + 1

print("\nO valor total da soma e de ",soma)
print("\nObigado por utilizar nosso sistema!")
os.system("pause")
